plugin/server/services/translate.js

```diff
@@ -69,6 +75,85 @@ module.exports = ({ strapi }) => ({
   async batchTranslateCancelJob(id) {
     return this.batchTranslateManager.cancelJob(id)
   },
+  async batchUpdate(params) {
```

Comment:
I believe doing this synchronously is not a good Idea as the translation might take a long time. 

I believe integrating this functionality into the BatchTranslateJob would be more coherent, but would probably require some refactoring.